### ISSUE TITLE
Incorrect command for launching notebook on AWS

### ISSUE THREAD

**Issue by JiaweiZhuang** [alanedelman/18.337_2017#3]
Following [infrastructure_guide.md](https://github.com/alanedelman/18.337_2017/blob/master/infrastructure_guide.md), I tried this command to launch a notebook but got an error:

```
julia> ;$(IJulia.notebook_cmd[1]) notebook list
ERROR: syntax: extra token "notebook" after end of expression
```

`IJulia.notebook_cmd[1]` simply prints the command and doesn't take additional arguments:

```
julia> IJulia.notebook_cmd[1]
"/home/ubuntu/.julia/v0.6/Conda/deps/usr/bin/jupyter"
```

So I use the traditional way to launch a notebook and it works fine:

```
$ /home/ubuntu/.julia/v0.6/Conda/deps/usr/bin/jupyter notebook --port 8080
```

**Comment by vchuravy** [comment]
Thanks. I probably should have make that clearer. `;` switches into shell mode in the REPL, but it seems that you can't copy on paste that.

```
run(`$(IJulia.notebook_cmd[1]) notebook list`)
```

Will work as well.

The next iteration off the AWS image will include Jupyter installed by default and not via Conda.

**Comment by vchuravy** [comment]
fixed in 19510098
